Makefile

```diff
@@ -2,8 +2,8 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 
 HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
-HANDLER_IMAGE_SUFFIX ?=
-HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME)$(HANDLER_IMAGE_SUFFIX)
+HANDLER_IMAGE_SUFFIX ?= "latest"
```

Comment:
'latest' default value was removed on purpose, since using ":latest" is not good practices, the idea is to use image un tagged, if you want to get the latest.

Some stuff related to "latest" https://blog.container-solutions.com/docker-latest-confusion

Reply:
correct me if I am wrong here but it you build without a tag you will get latest by default no?

Reply:
Wait you are right, so then we don't need explicit "latest" here, since docker is going to do the "latest" tagging for us.